src/main.js

```diff
@@ -58,27 +58,38 @@ const checkValidationOfValues = envSpecString => {
   }
 };
 
+//function that creates entries
+const newEntry = (envName, envType, envChoices, envDefaultVal) => {
```

Comment:
We should prefer using a class here instead of a function (we could name it `Entry`), based on the example here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#Class_declarations.

The class' constructor should take the same arguments as `newEntry`.

```javascript
const newEntry = new Entry('ADMIN_EMAIL', 'email', null, 'hello@example.com');  // We should prefer this
const oldEntry = newEntry('ADMIN_EMAIL', 'email', null, 'hello@example.com');  // We should NOT prefer this
```